examples/nested/values.yaml

```diff
@@ -0,0 +1,24 @@
+spire-server:
+  enabled: true
+  upstreamAuthority:
+    spire:
+      enabled: true
+      server:
+        address: spire-server.spire-root-server
```

Comment:
This value is duplicated on line 24, leading to the ability to get it wrong in one or both places.  This is the example that proves the design of the YAML values is wrong.

Reply:
Done